concrete-core/src/specification/engines/lwe_keyswitch_key_inplace_conversion.rs

```diff
@@ -0,0 +1,46 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::LweKeyswitchKeyEntity;
+
+engine_error! {
+    LweKeyswitchKeyInplaceConversionError for LweKeyswitchKeyInplaceConversionEngine @
+    LweDimensionMismatch => "The two keys must have the same lwe dimension.",
```

Comment:
You should check both the input and output LWE dimensions.

Reply:
Good catch thanks !